ApexOSA/filter.py

```
@@ -0,0 +1,271 @@
+        return value
+
+    def SetFilterOutput(self, State=False):
+        """
+        Sets the state of the internal optical switch
+        State can be a boolean or an integer:
+            - State = 0 or False : the filter output is not enabled (default)
+            - State = 1 or True : the filter output is enabled
+        """
+
+        if not isinstance(State, (int, bool)):
+            raise ApexError(APXXXX_ERROR_ARGUMENT_TYPE, "State")
```

Ruff (0.12.2)

`ApexError` may be undefined, or defined from star imports

(F405)